fjord/feedback/jinja2/feedback/android_thanks.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'generic_feedback' %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'android-feedback' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Firefox for Android Feedback') }}{% endblock %}
+
+{% block body_id %}android{% endblock %}
+
+{% block body %}
+  <div class="main">
+
+    {% block content %}
+      <div class="content">
+        <div class="deck">
+          <div class="card no-back" id="thanks">
+            <section>
+
+              <h1>
+                {% trans %}
+                  Thank you.
+                {% endtrans %}
+              </h1>
+              <div class="message heart">
+                <svg viewBox="0 0 100 110">
+                  <path d="M4.20152102,44.5344113 C-12.8987227,5.08506476 37.2648073,-17.6397688 50.6373929,18.0189693 C63.9576781,-17.6397683 114.277469,5.08506476 97.1772249,44.5344113 C91.8645859,55.9698934 52.2858248,83.7136852 50.5850925,90.6483352 C48.9013298,83.7202158 9.80833953,55.370549 4.20152102,44.5344113 Z" fill="#DE1920"></path>
+                </svg>
+                {% trans %}
+                  We're always working to make Firefox better.
+                  Rest assured that real people will look at your feedback 
+                  and do their very best to resolve your issue.
+                {% endtrans %}
```

Comment:
Ha! That'll do it!

I usually test feedback with "video never works. the screen is always black." It needs to be > 7 words long and I like to throw in "video" because that brings back a bunch because video is awesome.